docs/cartesian-product-v1.adoc

```diff
@@ -1,11 +1,14 @@
-:page-title: Old Cartesian product of all test parameters (deprecated in 1.6.0)
-:page-description: Extends JUnit Jupiter with `@CartesianProductTest`, a variant of parameterized tests that tests all combinations of its input
+:page-title: Cartesian Product of Parameters [removed in 2.0]
+:page-description: The JUnit 5 (Jupiter) extension `@CartesianTest` adds a variant of parameterized tests that executes a test for each possible combination of input arguments
```

Comment:
All mentions of `CartesianTest` should be `CartesianProductTest` in this document.

Reply:
Good catch, stupid copy-paste error.